app/js/components/manageApps/AddonList.jsx

```diff
@@ -9,74 +9,34 @@
  */
 import React from 'react';
 import { Link } from 'react-router';
+import SingleAddon from './SingleAddon.jsx';
 
 export const AddonList = ({ handleDownload, appList, openPage, openModal, updatesAvailable }) => {
+  console.log(updatesAvailable);
```

Comment:
You should remove all console.logs from your PR. They are for development/debugging and should be removed when ready for production.